package.json

```diff
@@ -1,15 +1,27 @@
 {
   "name": "@usabilla/js-styleguide",
-  "version": "1.0.0",
+  "version": "0.0.0-development",
   "description": "Usabilla JavaScript styleguide",
   "main": "index.js",
   "repository": "git@github.com:usabilla/js-styleguide.git",
   "author": "development@usabilla.com",
   "license": "MIT",
   "dependencies": {
```

Comment:
Shouldn't we separate the dependencies and devDependencies so we don't have to do it in every new generated projet? If I am not mistaken these are all devDependencies?